src/components/gcds-site-menu/gcds-site-menu.css

```diff
@@ -651,45 +1060,52 @@
     right: auto;
     width: 25%;
   }
-  :host([menu-desktop-layout*=sidebar]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] li {
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] li {
     position: relative;
   }
-  :host([menu-desktop-layout*=sidebar]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] li > [role=menuitem] {
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] li > [role='menuitem'] {
     padding: calc(1rem) calc(1.5rem * 3) calc(1rem) 0;
   }
-  :host([menu-desktop-layout*=sidebar]) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] li > [data-h2-submenu-trigger] {
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] [data-h2-menu-container] > [data-h2-menulist] li > [data-h2-submenu-trigger] {
     padding: calc(1rem) 1.5rem;
     top: 1px;
   }
-  :host([menu-desktop-layout*=sidebar]) [data-h2-menu] [data-h2-menu-container] [data-optional-left] {
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] [data-h2-menu-container] [data-optional-left] {
     margin: 1.5rem;
-   }
-  :host([menu-desktop-layout*=sidebar]) [data-h2-menu] [data-h2-menu-container] [data-optional-right] {
+  }
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] [data-h2-menu-container] [data-optional-right] {
     margin: 1.5rem;
-   }
+  }
 }
 
 @media screen and (min-width: 64em) {
-  :host([menu-desktop-layout*=sidebar]) [data-h2-menu] + ::slotted(div), :host([menu-desktop-layout*=sidebar]) [data-h2-menu] + ::slotted(main) {
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] + ::slotted(div),
+  :host([menu-desktop-layout*='sidebar']) [data-h2-menu] + ::slotted(main) {
     margin-left: 25%;
     overflow-x: hidden;
     width: 75%;
   }
 }
 
-
 @media screen and (min-width: 64em) {
-  :host([menu-desktop-layout*=sidebar][menu-alignment*=right]) [data-h2-menu] {
+  :host([menu-desktop-layout*='sidebar'][menu-alignment*='right']) [data-h2-menu] {
     top: 0;
     left: auto;
     right: 0;
   }
 }
 @media screen and (min-width: 64em) {
-  :host([menu-desktop-layout*=sidebar][menu-alignment*=right]) [data-h2-menu] + ::slotted(div), :host([menu-desktop-layout*=sidebar][menu-alignment*=right]) [data-h2-menu] + ::slotted(main) {
+  :host([menu-desktop-layout*='sidebar'][menu-alignment*='right']) [data-h2-menu] + ::slotted(div),
+  :host([menu-desktop-layout*='sidebar'][menu-alignment*='right']) [data-h2-menu] + ::slotted(main) {
     margin-left: auto;
     margin-right: 25%;
     overflow-x: hidden;
     width: 75%;
   }
 }
+
+@media screen and (max-width: 76.625em) {
```

Comment:
The only time I see this take affect is when the mobile drawer is open, the menu has a gap on the right. Is this the intended style for this menu?

Reply:
the site-menu sort of breaks out of the edge of the other containers without this, it only happens for a short time span, did this more or less to fix it during that tiny window